UI Updates & Formatting Merge

MegnogJ opened the pull request:
Update to UI Template & Formatting

MegnogJ committed:
Moved changes from old branch to new one

Moved files so that they would be up to date with main (too many conflicts)

michibatoryu committed:
Utilities class, mouseover method, color enum, etc...

MegnogJ committed:
Added image directory

MegnogJ committed:
Fixed HomePage.fxml

MegnogJ committed:
Added Vector Images to Images folder

MegnogJ committed:
Created signage page

MegnogJ committed:
Renamed menu image

MegnogJ committed:
updated screen class

MegnogJ committed:
created service request page

MegnogJ committed:
Merge branch 'MT_UtilityClass' into UI_MOJ

MegnogJ committed:
Revert "created service request page"

This reverts commit 745398f9838b3e253ee12b487c0fe30d1209325c.

arvirone committed:
test code

Code with test to make it so that when one button is hovered over another disappears (in HomePageController)

arvirone committed:
Drop down button that works with hidden feature

code in HomePageController

MegnogJ committed:
Coding in button functionality

arvirone committed:
pop out menu with two buttons on homepage

arvirone committed:
all five buttons included in pop out menu in HomePage

arvirone committed:
menuDropDownVisibilty function created to simplify code

arvirone committed:
update

arvirone committed:
menuDropDownVisibilty made to simplify code

MegnogJ committed:
Updated Home Page

MegnogJ committed:
Re-did Flower Request Page

MegnogJ committed:
Updated Signage Controller

arvirone committed:
Merge branch 'UI_MOJ' of https://github.com/CS-3733-D23-Team-E/Prototype-Two into UI_MOJ

MegnogJ committed:
Updated Home Page & Tested

MegnogJ committed:
Flower Request Cancel Button added

MegnogJ committed:
MealRequest created & worked on

Not working yet. Hope to be done by meeting/halfway through meeting tomorrow

DiyarJaafar committed:
map displays

all 6 six maps display pages and their controllers

arvirone committed:
Merge branch 'UI_MOJ' of https://github.com/CS-3733-D23-Team-E/Prototype-Two into UI_MOJ

arvirone committed:
Added code to make side menu buttons go to the page they correspond to

arvirone committed:
Added pictures to side menu buttons

MegnogJ committed:
Floor 1 & Floor 2 Map Display Changes

MegnogJ committed:
Merge branch 'mapDisplayForPath' into UI_MOJ

MegnogJ committed:
Merged In Map Pages

MegnogJ committed:
Reformatted & Themed All Map Pages

MegnogJ committed:
Clear Form Button Added

MegnogJ committed:
Merge branch 'main' into UI_MOJ

MegnogJ committed:
Merged in Main

KalticCodes commented on src/main/java/edu/wpi/teame/controllers/FloorOneMapNavController.java:
```diff
@@ -0,0 +1,30 @@
+    thirdFloorButton.setOnMouseClicked(event -> Navigation.navigate(Screen.FLOOR_THREE));
+    backButton.setOnMouseClicked(event -> Navigation.navigate(Screen.HOME));
+  }
+}
```
is having 6 MapNavControllers necessary? I feel like for iteration 1 we could probably improve this a little. Fine for now

KalticCodes commented on src/main/java/pathfinding/Floor.java:
```diff
@@ -0,0 +1,32 @@
+      default:
+        throw new NoSuchElementException("No such Floor found");
+    }
+  }
```
This file (and the others in this package) should have been moved to edu/wpi/teame/map from the previous pull request. Is this being reverted a result of a merge conflict or some other issue?

KalticCodes replied:
![image](https://user-images.githubusercontent.com/67293695/229636896-8959e662-e309-4238-adee-bcfef604ad38.png)
Adding to this: It seems your branch contains both the old and new locations for these files. Please delete the duplicates in the old location

KalticCodes commented on src/test/java/edu/wpi/teame/map/pathfinding/PathfindingTests.java:
```diff
@@ -2,12 +2,8 @@
-
-import edu.wpi.teame.map.pathfinding.AStarPathfinder;
-import edu.wpi.teame.map.pathfinding.AbstractPathfinder;
-import edu.wpi.teame.map.pathfinding.BFSPathfinder;
```
why did these imports get removed? Do these tests still pass like this?

KalticCodes reviewed (changes requested):
![image](https://user-images.githubusercontent.com/67293695/229637141-6fbb34e8-925d-4bc5-8cd3-20c0758ca3f7.png)
Error appears on run. Please review comments and make adjustments

MegnogJ committed:
Changes outlined in Pull Request comments

- Added getC() Getter
- re-added import functions to test file
- removed old pathfinding files

KalticCodes reviewed (approved):
Functionality confirmed, all comments addressed.

Pull request approved
